src/components/new-safe/create/AdvancedCreateSafe.tsx

```diff
@@ -0,0 +1,196 @@
+import { Container, Typography, Grid } from '@mui/material'
+import { useRouter } from 'next/router'
+
+import useWallet from '@/hooks/wallets/useWallet'
+import OverviewWidget from '@/components/new-safe/create/OverviewWidget'
+import type { TxStepperProps } from '@/components/new-safe/CardStepper/useCardStepper'
+import SetNameStep from '@/components/new-safe/create/steps/SetNameStep'
+import OwnerPolicyStep from '@/components/new-safe/create/steps/OwnerPolicyStep'
+import ReviewStep from '@/components/new-safe/create/steps/ReviewStep'
+import { CreateSafeStatus } from '@/components/new-safe/create/steps/StatusStep'
+import { CardStepper } from '@/components/new-safe/CardStepper'
+import { AppRoutes } from '@/config/routes'
+import { CREATE_SAFE_CATEGORY } from '@/services/analytics'
+import type { AlertColor } from '@mui/material'
+import type { CreateSafeInfoItem } from '@/components/new-safe/create/CreateSafeInfos'
+import CreateSafeInfos from '@/components/new-safe/create/CreateSafeInfos'
+import { type ReactElement, useMemo, useState } from 'react'
+import ExternalLink from '@/components/common/ExternalLink'
+import { HelpCenterArticle, LATEST_SAFE_VERSION } from '@/config/constants'
+import { type NewSafeFormData } from '.'
+import { type SafeVersion } from '@safe-global/safe-core-sdk-types'
+import AdvancedOptionsStep from './steps/AdvancedOptionsStep'
+
+const staticHints: Record<
```

Comment:
Wdyt about removing hints for this advanced flow or at least reusing the existing data to reduce code duplication?